packages/ui/src/components/Navigation/NavigationToggleButton/index.module.scss

```diff
@@ -11,19 +15,17 @@
   --utrecht-button-subtle-hover-border-width: 0;
   --utrecht-button-hover-scale: 1;
   --utrecht-button-focus-scale: 1;
-
   justify-content: flex-end;
   padding-inline-end: var(--utrecht-navigation-toggle-button-padding-inline-end);
+  margin-block-start: calc(-1 * var(--breakout-offset));
```

Comment:
Ik denk dat je de button-spacing tokens beter kunt gebruiken in plaats van een nieuw token aan te maken (--breakout-offset: 15px;). Dat maakt het dynamischer.

```css
  margin-block-start: calc(-1 * var(--utrecht-button-padding-block-end));
  margin-inline-end: calc(-1 * var(--utrecht-button-padding-inline-end));
```